### PR TITLE
Vinny/sr model implementation

### PR DESCRIPTION
**Pull request by vinayakSharm**
adding video 4x reconstruction module via VSR GAN. 

Model based on the paper: https://arxiv.org/pdf/2111.12704.pdf

(1) Added RealBasicVSR_reconstruction_module
(a) builder.py to help build the model and load it from s3
(b) realbasicvsr_x4.py has a 4x config dictionary

(2) Updated runner notebook to demonstrated full pipeline of trimming video, downsampling video with ffmpeg, super resolving video 4x using GAN

(3) minor updates to aux and labeler.

### CODE REVIEW

File: runner_notebooks/leto-demo.ipynb
Comment (PierceLovesee) [inline runner_notebooks/leto-demo.ipynb:1]: torch.nn should be included in either the existing requirments.txt, or an additional requirements.txt

Comment (PierceLovesee) [inline runner_notebooks/leto-demo.ipynb:1]: mmcv is not installing correctly as is.

Comment (vinayakSharm) [reply runner_notebooks/leto-demo.ipynb:1]: added.

Comment (vinayakSharm) [reply runner_notebooks/leto-demo.ipynb:1]: fixed

File: runner_notebooks/leto-demo.ipynb [hunk 1]
```diff
@@ -13,52 +13,182 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5ede7f32-1820-4001-be52-14643e618f5b",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "!pip install -r requirements_reconstruction.txt"
```
[the remark this answers is not in the document]
Comment (PierceLovesee) [reply runner_notebooks/leto-demo.ipynb:129]: ERROR: Failed building wheel for mmcv-full
  Running setup.py clean for mmcv-full
  Building wheel for filterpy (setup.py) ... done
  Created wheel for filterpy: filename=filterpy-1.4.5-py3-none-any.whl size=110474 sha256=3322935d7f9d72b222f8651df38068102f5d9f56660cc316a521a5bfc67f811b
  Stored in directory: /root/.cache/pip/wheels/ce/e0/ee/a2b3c5caab3418c1ccd8c4de573d4cbe13315d7e8b0a55fbc2
Successfully built filterpy
Failed to build mmcv-full
Installing collected packages: tensorboard-plugin-wit, lmdb, av, addict, tensorboard-data-server, setuptools, pyasn1-modules, ordered-set, oauthlib, mdurl, grpcio, cachetools, absl-py, requests-oauthlib, mmcv-full, markdown-it-py, markdown, google-auth, botocore, rich, model-index, google-auth-oauthlib, tensorboard, opendatalab, filterpy, boto3, openmim, facexlib, mmedit
  Attempting uninstall: setuptools
    Found existing installation: setuptools 59.3.0
    Uninstalling setuptools-59.3.0:
      Successfully uninstalled setuptools-59.3.0
  Running setup.py install for mmcv-full ... error
  error: subprocess-exited-with-error
  
  × Running setup.py install for mmcv-full did not run successfully.
  │ exit code: 1
  ╰─> [25 lines of output]
      /tmp/pip-install-b_e8xjmy/mmcv-full_f0729b79f4ab4a5a8c6df8ee733111f0/setup.py:6: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
        from pkg_resources import DistributionNotFound, get_distribution
      /opt/conda/lib/python3.7/site-packages/torch/cuda/__init__.py:52: UserWarning: CUDA initialization: Found no NVIDIA driver on your system. Please check that you have an NVIDIA GPU and installed a driver from http://www.nvidia.com/Download/index.aspx (Triggered internally at  /opt/conda/conda-bld/pytorch_1607370141920/work/c10/cuda/CUDAFunctions.cpp:100.)
        return torch._C._cuda_getDeviceCount() > 0

File: runner_notebooks/leto-demo.ipynb
[the remark this answers is not in the document]
Comment (vinayakSharm) [reply runner_notebooks/leto-demo.ipynb:1]: fixed

File: runner_notebooks/leto-demo.ipynb [hunk 2]
```diff
@@ -13,52 +13,182 @@
...
+      "Solving environment: done\n",
+      "\n",
+      "\n",
+      "==> WARNING: A newer version of conda exists. <==\n",
+      "  current version: 22.9.0\n",
+      "  latest version: 23.5.0\n",
+      "\n",
+      "Please update conda by running\n",
+      "\n",
+      "    $ conda update -n base -c defaults conda\n",
```
[the remark this answers is not in the document]
Comment (vinayakSharm) [reply runner_notebooks/leto-demo.ipynb:288]: fixed

File: runner_notebooks/leto-demo.ipynb [hunk 3]
```diff
@@ -13,52 +13,182 @@
...
+      "Executing transaction: done\n",
+      "Retrieving notices: ...working... done\n",
+      "\n",
+      "Note: you may need to restart the kernel to use updated packages.\n"
+     ]
+    }
+   ],
+   "source": [
+    "conda update -n base -c defaults conda\n",
+    "conda install pytorch==1.7.1 torchvision==0.8.2 torchaudio==0.7.2 cudatoolkit=10.1 -c pytorch"
```
[the remark this answers is not in the document]
Comment (vinayakSharm) [reply runner_notebooks/leto-demo.ipynb]: fixed

File: runner_notebooks/leto-demo.ipynb (hunk 1 above)
[the remark this answers is not in the document]
Comment (vinayakSharm) [reply runner_notebooks/leto-demo.ipynb]: fixed with a setup.sh script

**Pull request closed without merging**